### DISCUSSION TITLE
What commands are run when a system shutdown is issued?

### DISCUSSION THREAD

**Discussion by zinixyt** [itzg/docker-minecraft-server#1263] [Q&A] [answered]
I have a home server, with various services, and i only have it on when i want to use it.
So i wanted to know if when a system shutdown is issued, docker runs a save-all command on the server, to prevent world corruptions.
If not, i would like to know if there is a way to do that.

**Comment by itzg** [comment, marked as answer]
Yes, a container stop is handled by the server runner and performs a graceful stop of the server including world data saving.

**Reply by zinixyt** [reply]
Thanks!
